docs/pages/identity-governance/entra-id/getting-started.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Getting started with the Entra ID integration
+description: Describes how to set up the Teleport Entra ID integration in Teleport.
+labels:
+ - how-to
+ - identity-governance
+---
+
+
+This guide shows how to configure Entra ID integration in a guided configuration set up. 
+
+Teleport will generate a script that will configure your Entra ID tenant with the 
+properties required for the Teleport Entra ID integration.
+
+## Prerequisites
+
+- Teleport Identity Governance enabled for your Teleport cluster. Optionally, Identity Security should
+also be enabled if you choose to enable Identity Security integration.
```

Comment:
If we removed the identity security info from this guide, we can probably remove the prereq.